docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-hints-autodiscover.asciidoc

```diff
@@ -0,0 +1,211 @@
+[[hints-annotations-autodiscovery]]
+= Hints annotations based autodiscover
+
+beta[]
+
+NOTE: Make sure you are using {agent} 8.5+.
+
+Standalone {agent} supports autodiscover based on hints from the <<kubernetes-provider,provider>>.
```

Comment:
Should the fact that it is only supported for standalone be also a NOTE ? @gizas